pkg/kubectl/rollout_status.go

```diff
@@ -77,3 +83,23 @@ func (s *DeploymentStatusViewer) Status(namespace, name string, revision int64)
 	}
 	return fmt.Sprintf("Waiting for deployment spec update to be observed...\n"), false, nil
 }
+
+// Status returns a message describing daemon set status, and a bool value indicating if the status is considered done
+func (s *DaemonSetStatusViewer) Status(namespace, name string, revision int64) (string, bool, error) {
+	//ignoring revision as DaemonSets do not have history yet
```

Comment:
Wait, I was wrong. We support both DaemonSet update strategy (and Deployment as well).